gnomad/utils/annotations.py

```diff
@@ -893,3 +893,48 @@ def unphase_call_expr(call_expr: hl.expr.CallExpression) -> hl.expr.CallExpressi
         .when(call_expr.is_haploid(), hl.call(call_expr[0], phased=False))
         .default(hl.null(hl.tcall))
     )
+
+
+def region_flag_expr(
+    t: Union[hl.Table, hl.MatrixTable],
+    non_par: bool = True,
+    prob_regions: Dict[str, hl.Table] = None,
+) -> hl.expr.StructExpression:
+    """
+    Creates `region_flag` struct. Struct contains flags for problematic regions (i.e., LCR, decoy, segdup, and nonpar regions)
+
+    .. note:: No hg38 resources for decoy or self chain are available yet.
```

Comment:
I think it's good to have this note somewhere, and we can either keep it in this function or in the script(s) where this function is used to pass in regions. To me, it's nice to have it in a centralized location rather than scattered throughout various scripts, but I can see the argument from the other point of view

Reply:
OK, centralized sounds fine then